src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics.Test/ScenarioTests/AdlaAliasTests.cs

```diff
@@ -38,14 +38,24 @@ public void TestAdlaAccount()
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestAdlaAccountTiers()
         {
-            AdlaTestsBase.NewInstance.RunPsTest(true,
+            AdlaTestsBase.NewInstance.RunPsTest(false,
```

Comment:
This was a fix to remove creation of a WASB storage account, since this test doesn't need it to be created, it was making recording runs of this test take longer. Only the TestAdlaAccount() test needs to create a WASB storage account.